app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from datetime import date
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+        if visitor["vaccine"].get("expiration_date") < date.today():
+            raise OutdatedVaccineError
+        if not visitor.get("wearing_a_mask", False):
+            raise NotWearingMaskError
```

Comment:
The `NotWearingMaskError` should be raised with a descriptive message. According to the checklist, you should provide a message when raising exceptions. Consider using: `raise NotWearingMaskError('Visitor is not wearing a mask.')`.